src/HBaseClient.java

```diff
@@ -1964,6 +2057,19 @@ public String toString() {
   }
 
   /**
+  * Creates the META key to search for to locate the given key 
+  * with smallest timestamp (as opposed to largest in {@link #createRegionSearchKey}).
+  * @param table The table the row belongs to.
+  * @param key The key to search for in META.
+  */
+  private static byte[] createPreviousRegionSearchKey(final byte[] table,
+                                                      final byte[] key){
+    final byte[] meta_key = createRegionSearchKey(table, key);
+    meta_key[meta_key.length - 1] = '/';
```

Comment:
Add some comments for using "/", just like what createRegionSearchKey did for "meta_key[meta_key.length - 1] = ':'";